module-bsp/board/rt1051/bsp/eink/bsp_eink.cpp

```diff
@@ -188,25 +186,24 @@ status_t BSP_EinkInit()
 
     bsp::eink::eink_gpio_configure();
 
-    s_eink_lpspi_master_config.baudRate     = BSP_EINK_TRANSFER_WRITE_CLOCK;
-    s_eink_lpspi_master_config.bitsPerFrame = 8;
-    s_eink_lpspi_master_config.cpol         = kLPSPI_ClockPolarityActiveHigh;
-    s_eink_lpspi_master_config.cpha         = kLPSPI_ClockPhaseFirstEdge;
-    s_eink_lpspi_master_config.direction    = kLPSPI_MsbFirst;
-
-    s_eink_lpspi_master_config.pcsToSckDelayInNanoSec        = 1000000000 / s_eink_lpspi_master_config.baudRate;
-    s_eink_lpspi_master_config.lastSckToPcsDelayInNanoSec    = 1000000000 / s_eink_lpspi_master_config.baudRate;
-    s_eink_lpspi_master_config.betweenTransferDelayInNanoSec = 1000000000 / s_eink_lpspi_master_config.baudRate;
+    LPSPI_MasterGetDefaultConfig(&spi_config);
 
-    s_eink_lpspi_master_config.whichPcs           = kLPSPI_Pcs0;
-    s_eink_lpspi_master_config.pcsActiveHighOrLow = kLPSPI_PcsActiveLow;
+    spi_config.baudRate                      = BSP_EINK_TRANSFER_WRITE_CLOCK;
+    spi_config.bitsPerFrame                  = 8;
+    spi_config.cpol                          = kLPSPI_ClockPolarityActiveHigh;
+    spi_config.cpha                          = kLPSPI_ClockPhaseFirstEdge;
+    spi_config.direction                     = kLPSPI_MsbFirst;
+    spi_config.pcsToSckDelayInNanoSec        = 1000000000 / spi_config.baudRate;
```

Comment:
Maybe some constexpr for that value? I guess it's number of nanoseconds in second :smile:

Reply:
It is just copied from the fsl library, but I agree. Added constexpr value ;)